content/_getting-started/copy-and-export-flow.md

```diff
@@ -32,6 +32,8 @@ An exact copy of the flow will be created in a draft mode. Then you can rename a
 
 ## Export Flow
 
+### Export a single Flow
+
 You can export your flow from one workspace in to another. For example you would like to export your flow from development to production workspace. Please note that the flow can be successfully exported only if the components used in it are accessible in the target workspace.
```

Comment:
You can export your flow from one workspace into another. For example, you would like to export your flow from the development to the production workspace. Please note that the flow can be successfully exported only if the components used in it are accessible in the target workspace.